default.config.yml

```diff
@@ -215,6 +216,7 @@ extra_packages:
 extra_security_enabled: false
 
 drush_version: "master"
+drush_path: "{{ drupal_composer_install_dir }}/vendor/drush/drush/drush"
```

Comment:
This needs documentation for local codebases. If you're not using composer with drush as a dependency the ansible tasks will fail.